packages/components/src/index.ts

```diff
@@ -36,32 +36,52 @@ export {
     animations,
     variables,
     Button,
+    ButtonProps,
     ButtonPin,
```

Comment:
For the sake of consistency we should at least export default html button props(`React.ButtonHTMLAttributes<HTMLButtonElement>`) for ButtonPin component